### ISSUE TITLE
Uncaught RuntimeException status code -1073741819

### ISSUE THREAD

**Issue by sendtoomprakash** [wkhtmltopdf/wkhtmltopdf#3388]
Hi!
I got the error message as  **Fatal error: Uncaught exception 'RuntimeException' with message 'The exit status code '-1073741819'** Which not allowed me to generate the pdf report.

**Comment by alloylab** [comment]
is this still an issue on 0.12.5?

**Comment by iamomprakash** [comment]
Yes @alloylab still i have same error come while generating the pdf report from html.

**Comment by sendtoomprakash** [comment]
Yes @alloylab still i received the same error while generating the report from html content.

**Comment by Tomsgu** [comment]
Can you please post a minimal example with a command to generate it? So the issue can be reproduced.
What os do you have? How did you install the wkhtmltopdf?

**Comment by sendtoomprakash** [comment]
Ok @Tomsgu i will post the example.
I have Windows7 OS, and yes i have installed wkhtmltopdf.
